package.json

```diff
@@ -30,6 +30,10 @@
     "@types/requirejs": "^2.1.30",
     "@types/sinon": "^2.3.3",
     "@types/sinon-chai": "^2.7.28",
+    "babel-core": "^6.26.0",
+    "babel-loader": "^7.1.2",
+    "babel-polyfill": "^6.26.0",
```

Comment:
The docs lead me to believe that babel-polyfill, being a polyfill, needs to be a runtime dependency, not a dev dependency.  can you explain why it works as a dev dependency? https://babeljs.io/docs/usage/polyfill/

Reply:
when you specify babel-polyfill in webpack config, it imports the polifyill and includes it in the js bundle at compile time so that polyfill is available at run time.